testng-curl/azure-pipelines.yml

```diff
@@ -0,0 +1,35 @@
+# Maven
```

Comment:
I assume this is a duplicate file.
If yes please remove any one

Reply:
remove -1 file